cellbase-app/app/cloud/kubernetes/deploy.sh

```diff
@@ -224,15 +224,19 @@ function deployNginx() {
 }
 
 function deployMongodbOperator() {
-  NAME="mongodb-operator${NAME_SUFFIX}"
-  DATE=$(date "+%Y%m%d%H%M%S")
-  ./charts/mongodb-operator/fetch-mongodb-operator-files.sh
+  NAME="mongodb-community-operator${NAME_SUFFIX}"
+  helm repo add mongodb https://mongodb.github.io/helm-charts
+  helm repo update
+  MONGODB_OPERATOR_VERSION="${MONGODB_OPERATOR_VERSION:-v0.7.2}"
```

Comment:
Shouldn't this be version 0.7.3? It is also not being used in the "helm upgrade" below